src/simulation/monsters/low/Cyclops.ts

```diff
@@ -0,0 +1,59 @@
+import LootTable from '../../../structures/LootTable';
+import SimpleMonster from '../../../structures/SimpleMonster';
+import UncommonSeedDropTable from '../../subtables/CommonSeedDropTable';
+import HerbDropTable from '../../subtables/HerbDropTable';
+import { GemTable } from '../../subtables/RareDropTable';
+
+export const CyclopsTable = new LootTable()
+	.every('Big bones')
+
+	/* Weapons and armour */
+	.add('Black knife', [4, 13], 16)
+	.add('Steel chainbody', 1, 2)
+	.add('Iron 2h sword', 1, 2)
+	.add('Iron chainbody', 1, 2)
+	.add('Steel dagger', 1, 2)
+	.add('Steel mace', 1, 2)
+	.add('Steel sword', 1, 2)
+	.add('Steel battleaxe', 1, 2)
+	.add('Steel 2h sword', 1, 2)
+	.add('Steel longsword', 1, 2)
+	.add('Steel med helm', 1, 2)
+	.add('Black 2h sword', 1, 1)
+	.add('Mithril dagger', 1, 1)
+	.add('Adamant mace', 1, 1)
+	.add('Black sword', 1, 1)
+	.add('Black longsword', 1, 1)
+	.add('Black dagger', 1, 1)
+	.add('Adamant 2h sword', 1, 1)
+
+	/* Herbs */
+	.add(HerbDropTable, 1, 3)
+
+	/* Seeds */
+	.add(UncommonSeedDropTable, 1, 1)
+
+	/* Coins */
+	.add('Coins', [3, 102], 31)
+	.add('Coins', [5, 204], 10)
+
+	/* Gem drop table */
+	.add(GemTable, 1, 2)
+
+	/* Tertiary */
+	.tertiary(400, 'Long bone')
+	.tertiary(512, 'Clue scroll (hard)')
+	.tertiary(5013, 'Curved bone')
+
+	/* Catacombs tertiary (Average) */
+	.tertiary(275, 'Ancient shard')
+	.tertiary(412, 'Dark totem base')
+	.tertiary(412, 'Dark totem middle')
+	.tertiary(412, 'Dark totem top');
```

Comment:
This is completely wrong. You're gonna be dropping catacombs items everywhere. Remove this section entirely

Reply:
No I only coded the Cyclopes in Catacombs, and the once in the warriors guild is having all those totem pieces removed manually in that activity so that we can have both Cyclopes in game without having to specify location of the monsters. But this cyclopes is also a preparation for slayer.